code/game/objects/items/weapons/storage/deferred.dm

```diff
@@ -142,21 +143,21 @@
 
 /obj/item/weapon/storage/deferred/crate/saw
 	name = "infantry support crate"
-	desc = "a crate containing two L6 SAW light machine guns, and 640 rounds of 7.62mm ammunition"
-	icon_state = "weaponcrate"
-	initial_contents = list(/obj/item/weapon/gun/projectile/automatic/l6_saw = 2, /obj/item/ammo_magazine/ammobox/a762 = 8)
+	desc = "a crate containing two Pulemyot Kalashnikova light machine guns, and 640 rounds of 7.62mm ammunition"
```

Comment:
desc = "a crate containing two Pulemyot Kalashnikova light machine guns, and 800 rounds of 7.62mm ammunition"

Reply:
Our AK isn't actually an AK47, Its an AKMysteriousNumber onto which we can handwave whatever properties we desire.

In practical terms, this thing has 600 RPM and thusly an insane damage output, it doesn't also need more powerful ammo

Reply:
Or the AK-47 can have a weaker ammunition with the PK's output remaining the same. However you like it.